qa/sc_bootstrap.py

```diff
@@ -33,58 +34,37 @@ def setup_nodes(self):
     def sc_setup_chain(self):
         mc_node = self.nodes[0]
         sc_node_1_configuration = SCNodeConfiguration(
-            MCConnectionInfo()
+            MCConnectionInfo(address="ws://{0}:{1}".format(mc_node.hostname, websocket_port_by_mc_node_index(0)))
         )
         sc_node_2_configuration = SCNodeConfiguration(
-            MCConnectionInfo()
+            MCConnectionInfo(address="ws://{0}:{1}".format(mc_node.hostname, websocket_port_by_mc_node_index(0)))
         )
         sc_node_3_configuration = SCNodeConfiguration(
-            MCConnectionInfo()
+            MCConnectionInfo(address="ws://{0}:{1}".format(mc_node.hostname, websocket_port_by_mc_node_index(0)))
         )
         network = SCNetworkConfiguration(SCCreationInfo(mc_node, "1".zfill(64), 100, 1000),
                                          sc_node_1_configuration, sc_node_2_configuration, sc_node_3_configuration)
         self.sc_nodes_bootstrap_info = bootstrap_sidechain_nodes(self.options.tmpdir, network)
 
     def sc_setup_nodes(self):
-        return start_sc_nodes(self.number_of_sidechains, self.options.tmpdir)
-
-    def check_genesis_balances(self, sc_node, sidechain_id, expected_keys_count, expected_boxes_count, expected_wallet_balance):
-        print("Genesis checks for sidechain id {0}.".format(sidechain_id))
-        print("Checking that each public key has a box assigned with a non-zero value.")
-        response = sc_node.wallet_allPublicKeys()
-        public_keys = response["result"]["propositions"]
-        response = sc_node.wallet_allBoxes()
-        boxes = response["result"]["boxes"]
-        response = sc_node.wallet_balance()
-        balance = response["result"]
-        assert_equal(expected_keys_count, len(public_keys), "Unexpected number of public keys")
-        assert_equal(expected_boxes_count, len(boxes), "Unexpected number of boxes")
-        for key in public_keys:
-            target = None
-            for box in boxes:
-                if box["proposition"]["publicKey"] == key["publicKey"]:
-                    target = box
-                    assert_true(box["value"] > 0, "Non positive value for box: {0} with public key: {1}".format(box["id"], key))
-                    break
-            assert_true(target is not None, "Box related to public key: {0} not found".format(key))
-        print("Checking genesis balance.")
-        assert_equal(expected_wallet_balance, int(balance["balance"]), "Unexpected balance")
-        print("Total balance: {0}".format(json.dumps(balance["balance"])))
+        return start_sc_nodes(self.number_of_sidechain_nodes, self.options.tmpdir)
 
     def run_test(self):
         mc_nodes = self.nodes
         print "Number of MC nodes: {0}".format(len(mc_nodes))
         assert_equal(1, len(mc_nodes), "The number of MC nodes is grater than 1.")
```

Comment:
do we need this check? it was specified in `setup_nodes`

Reply:
Removed